src/locales/ko/dialogue.ts

```diff
@@ -1402,33 +1402,33 @@ export const PGMdialogue: DialogueTranslationEntries = {
   },
   "lance": {
     "encounter": {
-      1: "I've been waiting for you. Allow me to test your skill.",
-      2: "I thought that you would be able to get this far. Let's get this started."
+      1: "널 기다리고 있었다. 그 실력을 시험해보겠어.",
+      2: "여기까지 올 수 있을거라고 생각했다. 슬슬 시작해볼까."
     },
     "victory": {
-      1: "You got me. You are magnificent!",
-      2: "I never expected another trainer to beat me… I'm surprised."
+      1: "날 따라잡았군. 훌륭해!",
+      2: "다른 트레이너가 날 이길 거라곤 생각 못했는데… 놀랍군."
     },
     "defeat": {
-      1: "That was close. Want to try again?",
-      2: "It's not that you are weak. Don't let it bother you."
+      1: "근소하군. 다시 해볼까?",
+      2: "네가 약해서가 아니다. 신경쓰지 말도록."
     }
   },
   "karen": {
     "encounter": {
-      1: "I am Karen. Would you care for a showdown with my Dark-type Pokémon?",
-      2: "I am unlike those you've already met.",
-      3: "You've assembled a charming team. Our battle should be a good one."
+      1: "난 카렌! 내 악 타입 포켓몬과의 승부를 원하니?",
+      2: "난 네가 이전에 만났던 트레이너들과는 달라.",
+      3: "매력적인 팀을 꾸렸네. 우리의 승부 재밌겠는 걸."
     },
     "victory": {
-      1: "No! I can't win. How did you become so strong?",
-      2: "I will not stray from my chosen path.",
-      3: "The Champion is looking forward to meeting you."
+      1: "안돼! 내가 지다니. 너 어떻게 그렇게 강해졌어?",
```

Comment:
```suggestion
      1: "안돼! 내가 지다니. 너 어떻게 그렇게 강해졌어?(고민중)",
```
여기가 너무 좀... (저희가아는) 카렌스럽지 않은 말투라 생각좀해보겠습니다